src/parser/parser.py

```diff
@@ -0,0 +1,64 @@
+"""Parses a PDF document.
+"""
+import logging
+
+
+import slate
```

Comment:
Since this library must be installed in order for the parser to work, this PR should contain one of two things:
1) instructions for manually installing dependencies
2) encapsulation of dependencies using something like [pipenv][], [virtualenv][], or a [Dockerfile][]

[pipenv]: https://pipenv-fork.readthedocs.io/en/latest/
[virtualenv]: https://packaging.python.org/key_projects/#virtualenv
[Dockerfile]: https://runnable.com/docker/python/dockerize-your-python-application

Reply:
I'm using virtualenv